mozilla/bedrock
# Update Glean debugging docs (Fixes #13876)

## One-line summary

Updates Glean to favor using the newer console API debugging options.

## Issue / Bugzilla link

#13876

## Testing

1. Open http://localhost:8000/en-US/
2. In the web console, set `window.Glean.setLogPings(true)` to enable verbose ping logging. Setting `window.Glean.setLogPings(false)` should disable it again.
3. Set `window.Glean.setDebugViewTag('bedrock')` will send pings to https://debug-ping-preview.firebaseapp.com/. Note you can use any string ID here, so I might use something like `alex-bedrock`.
